api/info/service.go

```diff
@@ -229,9 +229,18 @@ func (i *Info) Peers(_ *http.Request, args *PeersArgs, reply *PeersReply) error
 	peers := i.networking.PeerInfo(args.NodeIDs)
 	peerInfo := make([]Peer, len(peers))
 	for index, peer := range peers {
+		benchedIDs := i.benchlist.GetBenched(peer.ID)
+		benchedAliases := make([]string, len(benchedIDs))
+		for idx, id := range benchedIDs {
+			alias, err := i.chainManager.PrimaryAlias(id)
+			if err != nil {
+				return fmt.Errorf("failed to get primary alias for chain ID %s: %v", id, err)
```

Comment:
Aside from this one linting nit I think this LGTM. Thanks for opening this!
```suggestion
				return fmt.Errorf("failed to get primary alias for chain ID %s: %w", id, err)
```